website/content/docs/nia/api/tasks.mdx

```diff
@@ -122,3 +206,24 @@ $ curl --request DELETE \
   "request_id": "9b23eea7-a435-2797-c71e-10c15766cd73"
 }
 ```
+
+## Task Object
+
+The task object is used by the Task APIs as part of a request or response. It represents the task's [configuration information](/docs/nia/configuration#task).
```

Comment:
Do you think it makes sense to add something about how these fields share the same required fields as what is presented in the `configuration information` ?

Something like:

```suggestion
The task object is used by the Task APIs as part of a request or response. It represents the task's [configuration information](/docs/nia/configuration#task) and shares the same required fields.
```

Reply:
Oh... if I understand you, I think what you're encountering is that if the Create Task API shares this table, it's missing information on which fields are required (and maybe also default values?). Let me know if I misunderstand. What are your thoughts on:

> The task object is used by the Task APIs as part of a request or response. It represents the task's configuration information.
> 
> When configuring this task object for the Create Task API, please refer to the [configuration docs](/docs/nia/configuration#task) for required fields and default values.

Thinking outloud... it might be nice to have the required/default value info on the same page so that users won't have to toggle.

I'm also open to other options, e.g.
A) splitting up and each API has its own table (thought it would be nice to share a table, but we don't have to if it doesn't make for a good ux)
B) add new columns for required and default value to the shared table

What are your thoughts?

Reply:
I think it makes sense for them to share an object (in this case a table). They do after all share an object in the API spec including the required fields. That is to say that whoever is consuming the response can also expect, at a minimum, that the fields required would be present.

With that in mind, I don't think it would be confusing to add a required column? Defaults might be nice too, since then when a user receives a response with set values they know they didn't set (i.e. buffer period values) then they understand why the values are present and set to the defaults.

Reply:
Oh, and yep! you are correct that my comment was in regards to this being a shared object between Create/Get

Reply:
Thanks! For posterity, we chatted offline:
- I will add a required and default value columns
- I won't edit the blurb above the table
- We don't think the new columns will cause confusion for users reading the Get Task API